evap/staff/tools.py

```diff
@@ -375,6 +379,30 @@ def remove_user_from_represented_and_ccing_users(user, ignored_users=None, test_
     return remove_messages
 
 
+def remove_inactive_participations(user, test_run=False):
+    remove_messages = []
+    evaluations = []
+    if user.is_active and user.can_be_marked_inactive_by_manager:
```

Comment:
For the check as described in the issue::
> Long inactive users are users who are inactive (or can_be_marked_inactive_by_manager) and [...]

I would expect that we need to have a condition like
```python3
if not user.is_active or user.can_be_marked_inactive_by_manager:
```
here, or am I missing something?